alf/environments/wrappers.py

```diff
@@ -27,44 +29,144 @@
 class FrameStack(gym.Wrapper):
     """Stack previous `stack_size` frames, applied to Gym env."""
 
-    def __init__(self, env, stack_size=4, channel_order='channels_last'):
+    def __init__(
+            self,
+            env,
+            stack_size=4,
+            channel_order='channels_last',
+            fields_to_stack=None,
```

Comment:
Perhaps you can use the similar way as common.image_scale_transformer to handle the field name to make it more general in the case of nested observation.
https://github.com/HorizonRobotics/alf/blob/d8464e3d16d62309f3e3b64d8a2f57fc21382cbc/alf/utils/common.py#L438